test/library/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -576,8 +576,8 @@ def test_2q_epg(self):
 
         self.assertGreater(cx_epg.value.n, 0.08 * 0.75)
 
-    def test_correct_1q_depolarization(self):
-        """Compute 2Q EPG with 1Q depolarization correction."""
+    def test_2q_epg_with_correction(self):
+        """Test if 2Q EPG with (1Q EPG) correction gets better (smaller) than one without it."""
```

Comment:
I suggest instead: """Check that 2Q EPG with 1Q depolarization correction gives a better (smaller)
        result than without the correction."""

Reply:
Thanks. Done in [15c2782](https://github.com/Qiskit/qiskit-experiments/pull/898/commits/15c27822bc93be0a80a3d3962f06261cf4d952e2)